### ISSUE TITLE
Link to developer site is broken

### ISSUE THREAD

**Issue by pizerg** [Z-WavePublic/libzwaveip#62]
The link on the readme www.zwavepublic.com/developer points to https://www.silabs.com/support/z-wave/developer a NOT FOUND location.

**Comment by rzr** [comment]
Yes it looks this project is no more maintained in the open... is there any plan for the future ?
